src/pages/favorites/Favorites.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { BsArrowRight } from "react-icons/bs";
+import { FiChevronLeft } from "react-icons/fi";
+import { HiOutlineMinusSm } from "react-icons/hi";
+import { Link } from "react-router-dom";
+import { useFavorites } from "../../stogre/addToLikes";
+import "./Favorites.css";
+const Favorites = () => {
+  const favoritesContext = useFavorites();
+  const favoriteProduct = favoritesContext.favorites;
+  const RemoveFavoriteHandler = (id) => {
+    const isFavorite = favoritesContext.itemIsFavorite(id);
+    if (isFavorite) {
+      return favoritesContext.removeFavorite(id);
+    }
+  };
+  return (
+    <main className="favorites-page">
+      <header className="header-of-favorites">
+        <Link key="1" to="/listofproducts">
+          <FiChevronLeft />
+        </Link>
+      </header>
+      <section className="favorites-container">
+        <h3>your favorites products</h3>
+        <ul className="favorite-items-container">
+          {favoriteProduct.map((product) => {
+            const { title, img1, price, id } = product;
+            return (
```

Comment:
if you are not doing a logic over the current iterating item. use object destructuring and simplify the return. so this can be:
```js
{favoriteProduct.map(({ title, img1, price, id }) => (
  <li ........
```